### DISCUSSION TITLE
Higher order derivatives

### DISCUSSION THREAD

**Discussion by Felipe-4** [jax-ml/jax#24228] [General] [answered]
Is it possible to take higher order derivatives with Jax ? ( say order 10) Are there any problems one should be aware of with higher order derivatives? I am wondering if it can be done automatically, in the sense that the effort to code order1 gradients is comparable to the effort to code order 10 gradients. I am thinking of a function with multiple variables.

**Comment by adam-hartshorne** [comment]
When you are talking such higher orders, wouldn't it be better to use Taylor Series Expansion approximation? Which you definitely can do very efficiently in JAX using the Jet operators.

**Reply by Felipe-4** [reply]
Hmmm nice, thanks. I read in some papers that AutoDiff for higher order derivatives wasn't well developed, but maybe they were outdated.

**Reply by adam-hartshorne** [reply, marked as answer]
This might help. They estimate higher order derivatives in JAX using Jet operators. 

Learning Differential Equations that are Easy to Solve
https://arxiv.org/abs/2007.04504
https://github.com/jacobjinkelly/easy-neural-ode
